pkg/virt-operator/resource/generate/components/secrets.go

```diff
@@ -28,6 +28,8 @@ const (
 	VirtApiCertSecretName           = "kubevirt-virt-api-certs"
 	VirtControllerCertSecretName    = "kubevirt-controller-certs"
 	CABundleKey                     = "ca-bundle"
+	caPodClusterLocal               = "pod.cluster.local"
```

Comment:
we could actually use atemplate string instead i think? 
`localPodDNStemplateString = "%s.%s.pod.cluster.local"